cryptoki/src/mechanism/mod.rs

```diff
@@ -986,6 +989,7 @@ impl Mechanism<'_> {
             Mechanism::AesKeyWrap => MechanismType::AES_KEY_WRAP,
             Mechanism::AesKeyWrapPad => MechanismType::AES_KEY_WRAP_PAD,
             Mechanism::AesGcm(_) => MechanismType::AES_GCM,
+            Mechanism::AesGcmMessage(_) => MechanismType::AES_GCM,
```

Comment:
Should there be a corresponding `MechanismType::AES_GCM_MESSAGE`?

Reply:
That is the odd part that the same mechanism is used with different parameter (and different functions), see the example in https://docs.oasis-open.org/pkcs11/pkcs11-spec/v3.1/os/pkcs11-spec-v3.1-os.html#_Toc111203302

If it would be more clear, we can do "phantom" mechanism for the message based parameter that would map to the same PKCS#11 mechanism though. But not sure if it would be easier, which is why I did not do that.

Reply:
I see, thanks for the precision! I think it's fine to keep it like this if that is consistent with the spec :)